src/components/GameResultNotice/NoticeItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import BaseNoticeItem from './BaseNoticeItem';
+
+type NoticeItemProps = {
+  rank: number;
+  nickname: string;
+  score: number;
+  calculation: string;
+  prize: string;
+};
+
+const NoticeItem: React.FC<NoticeItemProps> = ({
+  rank,
+  nickname,
+  score,
+  calculation,
+  prize,
+}) => {
+  return (
+    <BaseNoticeItem
+      leftContent={
+        <>
+          <div className="px-2 text-center">{rank}등</div>
+          <div className="px-2">{nickname}</div>
+          <div className="px-2">{score}</div>
+          <div className="px-2">{calculation}</div>
+        </>
```

Comment:
아이템 별 px-2 보다 gap-x-4 하는 게 어떨까
```suggestion
        <div className="flex items-center justify-center px-2 gap-x-4">
          <div>{rank}등</div>
          <div>{nickname}</div>
          <div>{score}</div>
          <div>{calculation}</div>
        </div>
```